### ISSUE TITLE
Why val and test set acc all zero?

### ISSUE THREAD

**Issue by ShangxuanWu** [jacobandreas/nmn2#20]
Hi,

I am currently training from your code, and the training loss/acc seems to be good. Yet the val and test set got all zero loss and acc. 

![image](https://cloud.githubusercontent.com/assets/12669087/24839475/f7645412-1d28-11e7-98f6-8259926ff609.png)

I think my paths and other things are configured as they should be. Has anyone met this issue before?

Thanks!

**Comment by jacobandreas** [comment]
This is expected---by default the code trains on both train and val,
validates on test-dev and tests on test. Both test-dev and test are
blind---there are no public labels, and the only way to get a score is to
upload to codalab. So we can't print out anything here. If you want to
change this behavior, look at line 169 in tasks/vqa.py

**Comment by ShangxuanWu** [comment]
Received. Thanks!

Shangxuan
